_posts/2021-09-29-boa-release-13.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Boa release v0.13"
+author: Boa Developers
+---
+
+Boa v0.13 is here! Boa is a JavaScript engine written in the Rust programming language. It makes it easy to embed a JS engine in your projects, and you can even use it from webassembly. See the [about](/about) page for more info.
+
+We currently support part of the language. In this release, our conformance has grown to 41.97% of the official ECMAScript Test Suite (Test262). We have closed 40 issues and merged 105 pull requests. You can check the full list of changes [here](https://github.com/boa-dev/boa/blob/v0.13/CHANGELOG.md).
+
+This release brings some new features, such as support for calling Rust closures from JavaScript to improve better interopability between JS and Rust.
+
+## ECMAScript language features
+
+[named capture groups](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Groups_and_Ranges) are now implemented and enabled.
+
+```javascript
+const RE_DATE = /(?<year>[0-9]{4})-(?<month>[0-9]{2})-(?<day>[0-9]{2})/;
+
+const matchObj = RE_DATE.exec("1999-12-31");
+const year = matchObj.groups.year; // 1999
+const month = matchObj.groups.month; // 12
+const day = matchObj.groups.day; // 31
+```
+
+This release brings support for the relative indexing method [String.prototype.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at). This makes it easier to fetch values at the end of an array or string instead of doing `str[str.length - 1]`.
+
+```javascript
+const sentence = "The quick brown fox jumps over the lazy dog.";
+let index = 5;
+console.log(
+  `Using an index of ${index} the character returned is ${sentence.at(index)}`
+);
+// expected output: "Using an index of 5 the character returned is u"
+```
+
+Other implemented language features include destructoring assignments, `replaceAll()`, `Object.values()`, `Object.keys()`, `Object.preventExtensions()`, `splice()`, `sort()`, spreading objects and more.
+
+## Boa API
+
+### Calling Rust closures from JavaScript
+
+In addition to registering Rust functions as global JavaScript functions, our API has been expanded to register Rust closures. [`register_global_closure`](https://github.com/boa-dev/boa/blob/master/boa/src/context.rs#L691) allows to capture variables in closures that can be called from javascript:
```

Comment:
We could clarify that only `Copy` closures are allowed and point them to the linked example to see how to move `Clone` types. Maybe a link to the corresponding issue explaining why this is needed.